Assets/Scripts/MRCCameraServer.cs

```diff
@@ -0,0 +1,173 @@
+
+public class MRCCameraServer: MonoBehaviour {
+
+    private TcpListener tcpListener; 
+    private Thread tcpListenerThread; 
+    private TcpClient connectedTcpClient;
+    
+    private OVRPose? calibratedCameraPose = null;
+
+    private OVRPose cameraPose = OVRPose.identity;
+
+    public void Start() {
+        tcpListenerThread = new Thread (new ThreadStart(ListenForIncommingRequests));         
+        tcpListenerThread.IsBackground = true;         
+        tcpListenerThread.Start();     
+    }
+
+    public void Update()
+    {
+        if (!calibratedCameraPose.HasValue)
+        {
+            if (!OVRPlugin.Media.GetInitialized())
+                return;
+            OVRPlugin.CameraIntrinsics cameraIntrinsics;
+            OVRPlugin.CameraExtrinsics cameraExtrinsics;
+            OVRPlugin.GetMixedRealityCameraInfo(0, out cameraExtrinsics, out cameraIntrinsics);
+            calibratedCameraPose = cameraExtrinsics.RelativePose.ToOVRPose();
+        }
+
+        // The receivedCameraPose is relative to the original calibrated pose, which is itself expressed in stage space.
+        OVRPose cameraStagePoseInUnits = calibratedCameraPose.Value * cameraPose;
+
+        // Converting position from meters to decimeters (unit used by Open Brush)
+        cameraStagePoseInUnits.position *= TiltBrush.App.METERS_TO_UNITS;
+
+        // Workaround to fix the OVRExtensions.ToWorldSpacePose() and OVRComposition.ComputeCameraWorldSpacePose()
+        // when computing the Mixed Reality foreground and background camera positions.
+        OVRPose headPose = OVRPose.identity;
+
+        Vector3 pos;
+        Quaternion rot;
+        if (OVRNodeStateProperties.GetNodeStatePropertyVector3(Node.Head, NodeStatePropertyType.Position, OVRPlugin.Node.Head, OVRPlugin.Step.Render, out pos))
+            headPose.position = pos;
+        if (OVRNodeStateProperties.GetNodeStatePropertyQuaternion(Node.Head, NodeStatePropertyType.Orientation, OVRPlugin.Node.Head, OVRPlugin.Step.Render, out rot))
+            headPose.orientation = rot;
+
+        OVRPose headPoseInUnits = OVRPose.identity;
+        headPoseInUnits.position = headPose.position * TiltBrush.App.METERS_TO_UNITS;
+        headPoseInUnits.orientation = headPose.orientation;
+
+        OVRPose stageToLocalPose = OVRPlugin.GetTrackingTransformRelativePose(OVRPlugin.TrackingOrigin.Stage).ToOVRPose();
+        OVRPose stageToLocalPoseInUnits = OVRPose.identity;
+        stageToLocalPoseInUnits.position = stageToLocalPose.position * TiltBrush.App.METERS_TO_UNITS;
+        stageToLocalPoseInUnits.orientation = stageToLocalPose.orientation;
+
+        OVRPose cameraWorldPoseInUnits = headPoseInUnits.Inverse() * stageToLocalPoseInUnits * cameraStagePoseInUnits;
+        OVRPose cameraStagePoseFix = stageToLocalPose.Inverse() * headPose * cameraWorldPoseInUnits;
+
+        // Override the MRC camera's stage pose
+        OVRPlugin.OverrideExternalCameraStaticPose(0, true, cameraStagePoseFix.ToPosef());
```

Comment:
This is the part that's actually fixing the Oculus MRC. If you prefer, I can open a separate PR with just this fix (separate from the "camera sever").